core/src/main/java/org/elasticsearch/index/store/Store.java

```diff
@@ -705,7 +702,7 @@ public String toString() {
     /**
      * Represents a snapshot of the current directory build from the latest Lucene commit.
      * Only files that are part of the last commit are considered in this datastrucutre.
-     * For backwards compatibility the snapshot might include legacy checksums that
+     * For backwards compatibility, the snapshot might include legacy checksums that
```

Comment:
This `,` is optional in English. I don't tend to include it most of the time so I wouldn't change this.